indico/modules/events/requests/controllers.py

```diff
@@ -182,6 +189,11 @@ def _checkProtection(self):
     def _process(self):
         if self.request.state not in {RequestState.pending, RequestState.accepted}:
             raise BadRequest
-        self.definition.withdraw(self.request)
-        flash(_('You have withdrawn your request ({0})').format(self.definition.title))
+        try:
+            self.definition.withdraw(self.request)
+        except RequestModuleError:
+            message = 'There was a problem withdrawing your request ({0}). Please contact support for further details.'
+            flash(_(message).format(self.definition.title), 'error')
```

Comment:
@meluru this doesn't work. the `_()` must be right around the string  so it can be extracted for translation

if you did it due to the line length:

    msg = _('...')

should make enough space available ;)